install/create_switch_metadata_csv.md

```diff
@@ -2,6 +2,15 @@
 
 This page provides directions on constructing the `switch_metadata.csv` file.
 
+## Prerequisites
+
+- The SHCD file for the system.
```

Comment:
Agreed. I think for now we have to say SHCD here because the rest of the page only talks about how to get the information from that. We'll need to include instructions on how to get the info from CCJ/Paddle first.